tests/search/test-class-search.php

```diff
@@ -2029,7 +2029,8 @@ public function test__maybe_alert_for_prolonged_query_limiting( $difference, $sh
 			->method( 'send_to_chat' )
 			->with( '#vip-go-es-alerts', $this->anything(), $expected_level );
 
-		$es->maybe_alert_for_prolonged_query_limiting();
+		// Surpress the trigger_error warning since it will always happens and causes execution to fail
+		@$es->maybe_alert_for_prolonged_query_limiting();
```

Comment:
PHPUnit can be told to expect exceptions (which is what eventually happens when the called code triggers an error) - that has the benefit of not using error supression but also asserting that the warning is created:

https://github.com/Automattic/vip-go-mu-plugins/blob/429dc63cffdf82fdbc9f663ce904fb10e09a46b0/tests/search/test-class-search.php#L1267-L1268